person.rb

```diff
@@ -1,7 +1,7 @@
 require_relative './corrector'
 
 class Person
-  attr_reader :id
+  attr_reader :id, :rentals
   attr_accessor :name, :age
```

Comment:
`rentals` should have both `Getter` and `Setter` attributes. Kindly add `rentals` to attr_accessor